### PR TITLE
update Navbar

### PR DESCRIPTION
**Pull request by Akwang-Deu**
# Engineers Checklist

## What this PR does?
- List of updates for features, enhancements, or bugfixes

## How can it be tested?
- Please provide step-by-step guide on how to test
- If web or mobile, screenshots will be very helpful in addition

# Where is it documented?
- List JIRA tickets

## Backend Updates
- [ ] Updated Postman Documentation
	- [ ] Requests
	- [ ] Examples
	- [ ] Tests
- [ ] Updated Tasks on JIRA
- [ ] Tested API updates in `dev` environment
- [ ] CI Checks Passed

### CODE REVIEW

**Review by Nataly-Wambui — APPROVED** [review]

**Review by Ivy-Wanjiku — DISMISSED** [review]

**Review by mercy-kantai — DISMISSED** [review]

**Review by Ivy-Wanjiku — DISMISSED** [review]

**Review by Nataly-Wambui — DISMISSED** [review]

**Review by Maureen-Gatweri — DISMISSED** [review]

File: taka-reverse/src/About-Us/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { BsMenuApp } from "react-icons/bs";
 const About = ()=>{
     return(
         
-        
+        <div id='About-Us'>
```
Comment (hunterachieng) [inline taka-reverse/src/About-Us/index.js:9]: You can just add the id in the already existing div. No need to create a new one

File: taka-reverse/src/Contacts/index.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ import { BsFillTelephoneFill } from "react-icons/bs";
 function Contact(){
     return(
         
+        <div id='contact'>
```
Comment (hunterachieng) [inline taka-reverse/src/Contacts/index.js:11]: Same here

File: taka-reverse/src/Partners/index.js [hunk 3]
```diff
@@ -2,6 +2,8 @@ import './index.css'
 const Partners =()=>{
     return(
         
+
+        <div id='Partners'>
         <div className='partners'>
```
Comment (hunterachieng) [inline taka-reverse/src/Partners/index.js:7]: Same here

**Review by Nataly-Wambui — DISMISSED** [review]

**Review by Ivy-Wanjiku — DISMISSED** [review]

**Review by Maureen-Gatweri — DISMISSED** [review]

**Review by mercy-kantai — DISMISSED** [review]

File: taka-reverse/src/Partners/index.js [hunk 4]
```diff
@@ -2,7 +2,9 @@ import './index.css'
 const Partners =()=>{
     return(
         
-        <div className='partners'>  
+
+        
+        <div id='Partners'>
```
Comment (hunterachieng) [inline taka-reverse/src/Partners/index.js:7]: I meant this 
```
<div id='partners' className='partners'>
```
They can work together. You should not have an Id starting with a capital letter

**Review by Nataly-Wambui — DISMISSED** [review]

**Review by mercy-kantai — APPROVED** [review]

**Review by Maureen-Gatweri — APPROVED** [review]

**Review by Maureen-Gatweri — APPROVED** [review]

**Review by Ivy-Wanjiku — APPROVED** [review]

**Review by mercy-kantai — APPROVED** [review]

**Review by Maureen-Gatweri — APPROVED** [review]

**Review by Nataly-Wambui — APPROVED** [review]

**Review by hunterachieng — APPROVED** [review]

**Pull request merged**